### DISCUSSION TITLE
Notification options to disable "your requested export" emails

### DISCUSSION THREAD

**Discussion by codemicro** [outline/outline#3934] [Ideas]
Hey!

I use the API to backup my self-hosted instance by creating an export each night, which also ends up sending me a pointless email in the process.

I was wondering if a notification option to prevent these emails being sent would be considered, to go alongside the existing "document published/updated/etc" notifications.

If so, I'll (attempt, since I'm not especially fluent with TS) to make a PR for it.

Have a good one :)

**Comment by tommoor** [comment]
Sounds good, we'd accept a PR with this functionality

**Reply by codemicro** [reply]
Awesome!

**Reply by tommoor** [reply]
Possible alternative might be to allow a boolean `emailNotification: false` or similar into the export API request to suppress the notif
